linalg/sparsemat.cpp

```diff
@@ -28,6 +28,23 @@ namespace mfem
 
 using namespace std;
 
+#ifdef MFEM_USE_CUDA
+int SparseMatrix::SparseMatrixCount = 0;
+cusparseHandle_t SparseMatrix::handle;
```

Comment:
I think these should at least be `private static` values inside the `SparseMatrix` class, or is there any reasons for them to be global variables?

Reply:
They are private variables inside ``SparseMatrix``. See sparsemat.hpp:L93,99

Reply:
Oh it's to do the initialization?